### DISCUSSION TITLE
Support migrating linkNewTab config from Redactor

### DISCUSSION THREAD

**Discussion by yumpdevs** [craftcms/ckeditor#198] [Ideas] [answered]
I noticed the `linkNewTab` config in Redactor is not compatible with CKEditor during the migration. It's a useful option for editors to set a link to open in new tab (target="_blank") or not, and would be great if that can be migrated over.

I found a Discord discussion for how to achieve something similar in CKEditor: https://discordapp.com/channels/456442477667418113/545074396457336835/1221878268534329404. Is it possible for the CKEditor plugin to support this migration?

**Comment by brandonkelly** [comment, marked as answer]
CKEditor’s link decorator support isn’t quite the same thing – you have to manually enable it for each link. So that’s why the `ckeditor/convert` command will just warn you that `linkNewTab` isn’t supported.
